googletest/src/gtest.cc

```diff
@@ -3259,6 +3259,8 @@ static const char* GetAnsiColorCode(GTestColor color) {
       return "2";
     case GTestColor::kYellow:
       return "3";
+    case GTestColor::kDefault:
```

Comment:
String

Reply:
Not sure what you are referring to here.